pkg/utils/utils.go

```diff
@@ -0,0 +1,18 @@
+package utils
```

Comment:
Could we use a more descriptive package name than "utils"?

Reply:
removed with some more refactoring